Maquetado/scss/grid.scss

```diff
@@ -0,0 +1,86 @@
+.tech-container-grid {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
+  grid-template-rows: auto;
+  grid-template-areas:
+    ". tech-a tech-b tech-c ."
+    ". tech-a tech-d tech-e .";
+  margin-bottom: 70px;
+}
+
+.tech-1 { 
```

Comment:
we should use a name related to the tech and not to the order, because we may need to for example make react small and make vue big and the numbers would be a mess.